examples/oneview_event.yml

```diff
@@ -0,0 +1,51 @@
+###
+# Copyright (2016) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+---
+- hosts: all
+  vars:
+    config: "{{ playbook_dir }}/oneview_config.json"
+  tasks:
+    - name: Create a basic Event
+      oneview_event:
+        config: "{{ config }}"
+        state: present
+        data:
+          description: "This is a very simple test event"
+          eventTypeID: "hp.justATest"
+          eventDetails: [{ eventItemName: "ipv4Address", eventItemValue: "198.51.100.5", isThisVarbindData: "false", varBindOrderIndex: -1}]
```

Comment:
That yaml syntax is valid, but usually we follow this other code style for lists and dicts, since we believe it makes the example more readable: 

```yaml
eventDetails:
  - eventItemName: ipv4Address
    eventItemValue: 198.51.100.5
    isThisVarbindData: false
    varBindOrderIndex: -1
```
The same is applicable for `serviceEventDetails` (line 39) and the other `eventDetails` (line 49) on this file.